go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/gravitational/teleport
 
-go 1.22.0
-
-toolchain go1.22.5
+go 1.22.5
```

Comment:
Go does that now, I assume because it might as well "simplify" if the go and toolchain directives point to the same version.

Fighting the toolchain seems pointless, so I let it be.